docs/content/latest/manage/backup-restore/point-in-time-restore-ycql.md

```diff
@@ -0,0 +1,268 @@
+            1223 |   Lucille Ball | Operations |  70000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (4 rows)
+    ```
+
+### Restore from a relative time
+
+In addition to restoring to a particular timestamp, you can also restore to a relative time, such as "ten minutes ago". In this example, you'll delete some data from the existing `employees` table, then restore the state of the database to what it was five minutes prior.
+
+When you specify a relative time, you can specify any or all of _days_, _hours_, and _minutes_. For example:
+
+* `"5m"` to restore from five minutes ago
+* `"1h"` to restore from one hour ago
+* `"3d"` to restore from three days ago
+* `"1h 5m"` to restore from one hour and five minutes ago
+
+**Careful!** If you specify a time prior to when you created the table, the restore will leave the table intact, but empty.
+
+1. Wait at least five minutes after you complete the steps in the previous section. This is so that you can easily use a known relative time for the restore.
+
+1. From the YCQL shell, remove employee 1223 from the table:
+
+    ```sql
+    ycqlsh:pitr> delete from employees where employee_no=1223;
+
+    ycqlsh:pitr> select * from employees;
+    ```
+
+    ```output
+    employee_no |      name      | department | salary 
+    -------------+----------------+------------+--------
+            1224 | John Zimmerman | Sales      |  60000
+            1221 | John Smith     | Marketing  |  50000
+            1222 | Bette Davis    | Sales      |  55000
+
+    (3 rows)
+    ```
+
+1. At a terminal prompt, restore the snapshot you created earlier:
+
+    ```sh
+    $ bin/yb-admin restore_snapshot bb5fc435-a2b9-4f3a-a510-0bacc6aebccf minus "5m"
+    ```
+
+1. Verify the restoration is in `RESTORED` state:
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                     State
+    bb5fc435-a2b9-4f3a-a510-0bacc6aebccf  COMPLETE
+    Restoration UUID                      State
+    bd7e4e52-b763-4b95-87ce-9399e1ac206e  RESTORED
+    ```
+
+1. Verify the data is restored, and employee 1223 is back:
+
+    ```sql
+    ycqlsh:pitr> select * from employees;
+    ```
+
+    ```output
+     employee_no | name           | department | salary
+    -------------+----------------+------------+--------
+            1223 |   Lucille Ball | Operations |  70000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (4 rows)
+    ```
+
+## Limitations
+
+This is a BETA feature, and is in active development. Currently, you can **restore data only**. The feature doesn't support metadata; in other words, it won't currently roll back operations such as CREATE, ALTER, TRUNCATE, and DROP TABLE.
```

Comment:
Do we want to say "metadata / schema changes"

Reply:
Maybe reword to "in other words, you currently shouldn't roll back beyond operations ... because it is unsupported."

Reply:
Updated!